INSTALL.md

```diff
@@ -179,11 +179,24 @@ the FFTW3 threading library libfftw3_threads (or libfftw3_omp) is required.
   and `LIBS += -L${LIBXSMM_DIR}/lib -lxsmmf -lxsmm -ldl`
 - LIBSMM is not used if LIBXSMM is enabled.
 
-### 2j. CUDA (optional, improved performance on GPU systems)
+### 2j. GPU support (optional, improved performance on GPU systems)
 
-- Specify OFFLOAD_CC (e.g. `OFFLOAD_CC = nvcc`) and
-  OFFLOAD_FLAGS (e.g. `OFFLOAD_FLAGS = -O3 -g -w --std=c++11`) variables.
-  Remember to include the support for the C++11 standard.
+By default CUDA or HIP support are enabled, imply that all cp2k code that
```

Comment:
I think, we should keep CPU-only as the default.
To enable GPU support the user should have to set either `__OFFLOAD_CUDA` or `__OFFLOAD_HIP`.

Reply:
I also thought it would be better.